testsupport/init.go

```diff
@@ -53,7 +54,12 @@ func WaitForDeployments(t *testing.T) wait.Awaitilities {
 
 		apiConfig, err := clientcmd.NewDefaultClientConfigLoadingRules().Load()
 		require.NoError(t, err)
-		kubeconfig, err := clientcmd.NewDefaultClientConfig(*apiConfig, &clientcmd.ConfigOverrides{}).ClientConfig()
+		configOverrides := clientcmd.ConfigOverrides{
+			ClusterDefaults: api.Cluster{
+				InsecureSkipTLSVerify: true,
```

Comment:
Do we want to make this an option we can pass make instead of hardcoding to true? What if we want to test security?

Reply:
+1

It seems that disabling the TLSVerify doesn't work on openshift CI at least. If this configuration works on your cluster then maybe we could introduce an env variable to optionally turn it off? 

Something like:
```
make test-e2e DISABLE_KUBE_CLIENT_TLS_VERIFY=true
````
and keep the tls verification on by default, as it is now.

Reply:
Nice! Extracting the config logic builder into a function is what I was thinking as well! 👍 

Should we add few lines to document this new variable, maybe adding a know issues/workarounds section here: https://github.com/codeready-toolchain/toolchain-e2e/blob/master/CRC.adoc 
reporting the tls failure and the workaround of using this env variable ? 

If it's not specific to CRC maybe we could add this new section in the main README.

WDYT?

Reply:
I like the idea, let me work on this! 👍🏾

Reply:
@mfrancisc  I've added a note in the README's `Running End-to-End Tests`. WDYT?